### ISSUE TITLE
Remote Support does not work in the latest versions of Nvda Alfa

### ISSUE THREAD

**Issue by gdata1** [nvaccess/nvda#8714]
#### Have you tried any other versions of NVDA?
Hello. In the latest versions of NVDA, the add-on does not work when enabled NVDA gives an error and everything.

NVDA version alpha-15968,10635ac2
log.
INFO - __main__ (10:09:33.137):
Starting NVDA
INFO - core.main (10:09:33.247):
Config dir: C:\Users\nikit\AppData\Roaming\nvda
INFO - config.ConfigManager._loadConfig (10:09:33.247):
Loading config: C:\Users\nikit\AppData\Roaming\nvda\nvda.ini
INFO - core.main (10:09:33.496):
NVDA version alpha-15968,10635ac2
INFO - core.main (10:09:33.496):
Using Windows version 10.0.17751 workstation
INFO - core.main (10:09:33.496):
Using Python version 2.7.15 (v2.7.15:ca079a3ea3, Apr 30 2018, 16:22:17) [MSC v.1500 32 bit (Intel)]
INFO - core.main (10:09:33.496):
Using comtypes version 1.1.3
INFO - core.main (10:09:33.496):
Using configobj version 5.1.0 with validate version 1.0.1
INFO - external:synthDrivers.RHVoice.SynthDriver.__init__ (10:09:33.709):
Using RHVoice version 0.7.0
INFO - synthDriverHandler.setSynth (10:09:33.714):
Loaded synthDriver RHVoice
INFO - core.main (10:09:33.714):
Using wx version 4.0.3 msw (phoenix) wxWidgets 3.0.5
INFO - brailleInput.initialize (10:09:33.721):
Braille input initialized
INFO - braille.initialize (10:09:33.723):
Using liblouis version 3.7.0
INFO - braille.BrailleHandler.setDisplayByName (10:09:33.723):
Loaded braille display driver noBraille, current display has 0 cells.
WARNING - core.main (10:09:33.792):
Java Access Bridge not available
INFO - _UIAHandler.UIAHandler.MTAThreadFunc (10:09:33.802):
UIAutomation: IUIAutomation6
ERROR - globalPluginHandler.initialize (10:09:34.621):
Error initializing global plugin <class 'globalPlugins.remoteClient.GlobalPlugin'>
Traceback (most recent call last):
  File "globalPluginHandler.pyc", line 32, in initialize
  File "C:\Users\nikit\AppData\Roaming\nvda\addons\remote\globalPlugins\remoteClient\__init__.py", line 56, in __init__
  File "C:\Users\nikit\AppData\Roaming\nvda\addons\remote\globalPlugins\remoteClient\__init__.py", line 104, in create_menu
AttributeError: 'MenuItem' object has no attribute 'SetCheckable'
INFO - core.main (10:09:34.759):
NVDA initialized

**Comment by leonardder** [comment]
Closing. Please post your concerns in https://github.com/NVDARemote/NVDARemote/issues/176

**Comment by Brian1Gaff** [comment]
You might wish to read back through messages in many of the lists about this 
problem with the add on. I believe there is a patched version out there, but 
I know that internal politics are involved here which NV Access are not 
party to, as this was a separate project funded by  user donation 
separately. Therefore, until the position is clarified, the patched version 
will work, but will fail later on when some new code is put into nvda.

Joseph knows much about this, but I only know what I've seen.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----
